x/txfees/keeper/txfee_filters/arb_tx.go

```diff
@@ -49,6 +51,9 @@ func (m AffiliateSwapMsg) TokenOutDenom() string {
 
 var _ poolmanagertypes.SwapMsgRoute = AffiliateSwapMsg{}
 
+type CrosschainSwap struct {
```

Comment:
I am curious, what is the use of this struct? 
maybe a comment would be useful

Reply:
ahh, it's not being used rn, so we can remove it (though we should when we check the XCS contract in the future)

Reply:
thanks for the review, btw! :)

Reply:
sure no problems :)